.npmignore

```diff
@@ -1,6 +0,0 @@
-.babelrc
-.eslint*
-.gitignore
-.nvm
-.travis*
-shrinkwrap.yaml
```

Comment:
Not needed because we have shifted to opting into which files will be packaged: see npm run dist

Reply:
Looking at "npm run dist" ...